sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/externalrules/CxxExternalRuleRepository.java

```diff
@@ -48,20 +50,29 @@ public CxxExternalRuleRepository(RulesDefinitionXmlLoader xmlRuleLoader, Setting
   @Override
   public void define(Context context) {
     NewRepository repository = context.createRepository(KEY, CxxLanguage.KEY).setName(NAME);
+    
     xmlRuleLoader.load(repository, getClass().getResourceAsStream("/external-rule.xml"), "UTF-8");
-
     for (String ruleDefs : settings.getStringArray(RULES_KEY)) {
       if (StringUtils.isNotBlank(ruleDefs)) {
         try {
           xmlRuleLoader.load(repository, new StringReader(ruleDefs));
         } catch (Exception ex) {
-          CxxUtils.LOG.info("Cannot Load XML '{}'", ex.getMessage());
+          CxxUtils.LOG.info("Cannot load rules XML '{}'", ex.getMessage());
+        }
+      }
+    }
+
+    SqaleXmlLoader.load(repository, "/com/sonar/sqale/cxx-model.xml");
```

Comment:
with this i think we might want to split the models so its easier to review a improve. i might try to do that at some point